examples/peripherals/usb/tusb_cdc_ecm/README.md

```diff
@@ -0,0 +1,6 @@
+# TinyUSB CDC-ECM Example
+
+Demonstrates how to use the ESP-IDF Framework API to perform a ethernet connection via USB.
+Afterwards two TCP Servers are started, which are listen on the Ports 3333 and 1234.
+In the folder tcp_client, you can find a tcp client written in python to test the communication.
+The client opens two sockets to address both ports. This shall demonstrate, that multiple ports can be served at once.
```

Comment:
Please add that DHCP is configured by default. User must have DHCP server to configure IP address.
Mention how to assign static IP